### DISCUSSION TITLE
Organization Member Search

### DISCUSSION THREAD

**Discussion by 2096779623** [community/community#18268] [Other Feature Feedback, Questions, & Ideas]
When searching for members in an organization with a large number of people (e.g. : [Epic Games](https://github.com/orgs/EpicGames/people)), the search efficiency is very slow, and only two strings can be searched.When it is greater than two strings, the result of the following picture will be obtained:

![image](https://user-images.githubusercontent.com/57583560/173075114-a1dbd66e-13d7-47b0-945c-a5f5f7e6e146.png)

![image](https://user-images.githubusercontent.com/57583560/173075188-41a24a5f-adf7-4711-9ac2-51415fb9b584.png)

How many strings can be searched depends on the number of people in the organization.

**Comment by MrKrishnaAgarwal** [comment]
Go to this website- https://github.com/orgs/EpicGames/people
<img width="956" alt="image" src="https://user-images.githubusercontent.com/100597998/173173255-1426fd05-1782-4b1a-91bc-e8a47c318509.png">

**Reply by 2096779623** [reply]
![image](https://user-images.githubusercontent.com/57583560/173183533-3f7bd08a-ca4c-4043-8136-816a14699cc9.png)

Same result.
